linuxrc.c

```diff
@@ -1278,6 +1278,13 @@ void lxrc_init()
 
   util_update_kernellog();
 
+  if(config.loghost)
+  {
+    char cmd[40];
```

Comment:
Only 13 characters are left for the loghost. Consider that even an IP address may be longer than that.